### ISSUE TITLE
Add content to Git tutorial

### ISSUE THREAD

**Issue by sharmaadityaHQ** [sharmaaditya570191/git-basics-tutorial#1]
Please follow the steps:-
**1.** Create a new text(.txt) file in the Git-Experiences folder.
**2.** You can add any interesting trick or fact about Git or whatever knowledge you have gained about Git till now. You can also share problems that you faced while working with Git along with their solution (if found).
**3.** Add appropriate commit message to describe your change.

**Comment by nazmariam** [comment]
Hi @sharmaaditya570191 could you assign me for this issue please?

**Comment by sharmaadityaHQ** [comment]
Yes please go ahead @nazmariam . Everyone can work on this issue as my aim is to collect various git experiences.
